packages/@n8n/db/src/index.ts

```diff
@@ -1 +1,14 @@
-export {};
+export {
```

Comment:
Would it make sense to split the exports by "scope", so the consumer would do:
```ts
import { WithTimestamps } from '@n8n/db'
import { generateNanoId } from '@n8n/db/utils'
```

Just wondering

Reply:
I like it! But I'm following the example with the other namespaced packages like `@n8n/di` and `@n8n/config` that export everything from root, so let's keep like this for now?